web/crux-ui/src/pages/nodes.tsx

```diff
@@ -50,10 +50,11 @@ const NodesPage = (props: NodesPageProps) => {
       id: message.nodeId,
       name: '',
       description: '',
+      type: 'docker',
       address: message.address,
       status: message.status,
       createdAt: new Date().toUTCString(),
-    }
+    } as DyoNode
```

Comment:
why?

Reply:
Default value if it is not selected yet.